gocd/bci.gocd.yaml

```diff
@@ -297,3 +335,22 @@ pipelines:
               osc -A https://api.suse.de/ api "/build/$PRJ/_result?view=summary&repository=images" | grep "result project" | grep 'code="published" state="published">' && echo PUBLISHED
             done
 
+  SLE_BCI_15SP5.RepoPublisher:
+    group: BCI
+    lock_behavior: unlockWhenFinished
+    materials:
+      git:
+        git: https://github.com/openSUSE/openSUSE-release-tools.git
+    environment_variables:
+        OSC_CONFIG: /home/go/config/oscrc-totest-manager
+        BCI_TOKEN: '{{SECRET:[opensuse.secrets][BCI_TOKEN]}}'
+    stages:
+    - Run:
+        approval: manual
```

Comment:
is the intention to switch that to automatic at a later point in time? I am confused on the bigger picture

Reply:
Yes